tests_v2/test_cli_notification_reopen_no_blank_window.py

```
@@ -0,0 +1,299 @@
+            return candidate, bundle_id
+
+    raise cmuxError(f"Could not locate a cmux app bundle next to CLI: {cli}")
+
+
+def _run_cli(cli: str, args: list[str]) -> str:
+    env = dict(os.environ)
+    env.pop("CMUX_WORKSPACE_ID", None)
+    env.pop("CMUX_SURFACE_ID", None)
+    env.pop("CMUX_TAB_ID", None)
+
+    proc = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)